src/Sulu/Component/HttpCache/EventSubscriber/KernelSubscriber.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\EventSubscriber;
+
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Sulu\Component\HttpCache\StructureCacheHandlerInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\Event\PostResponseEvent;
+use Sulu\Component\HttpCache\HandlerInterface;
+
+/**
+ * Listen to the content mapper and invalidate structures
+ */
+class KernelSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var Sulu\Component\HttpCache\StructureCacheHandlerInterface
+     */
+    private $handler;
+
+    /**
+     * @var Sulu\Component\Content\StructureInterface
```

Comment:
I don't think that would be accepted by the phpcsfixer and it is not the current practice in Symfony.